File: java64/nivel4/exer41.java
import javax.swing.JOptionPane;

public class exer41 {
    public static void main(String[] args) {
        int idade = Integer.parseInt(JOptionPane.showInputDialog("Informe a idade"));

        if (idade >= 5 && idade <= 7) JOptionPane.showMessageDialog(null, "Infantil A = 5 a 7 anos");
        if (idade >= 8 && idade <= 11) JOptionPane.showMessageDialog(null, "Infantil B = 8 a 11 anos");
        if (idade >= 12 && idade <= 13) JOptionPane.showMessageDialog(null, "Juvenil A = 12 a 13 anos");
        if (idade >= 14 && idade <= 17) JOptionPane.showMessageDialog(null, "Juvenil B = 14 a 17 anos");
        if (idade >= 18) JOptionPane.showMessageDialog(null, "Adultos = Maiores de 18 anos");
    }
}

File: pyhton64/nivel4/exer41.py
idade = int(input('Idade: '))

if idade == 5 and idade <= 7:
    print('Infantil A = 5 a 7 anos ')
elif idade > 7 and idade <= 11:
    print('Infantil B = 8 a 11 anos ')
elif idade > 11 and idade <= 13:
    print('Juvenil A = 12 a 13 anos ')
elif idade > 13 and idade <= 17:
    print('Juvenil B = 14 a 17 anos ')
elif idade > 17:
    print('Adultos = Maiores de 18 anos ')

